src/service/Commands/ShutdownCommand.cs

```diff
@@ -9,24 +9,16 @@
 
 namespace ChessBuddies.Commands
 {
-    public class ShutdownCommand : AdminCommand
+    public class ShutdownCommand : ModuleBase<SocketCommandContext>
     {
-        private readonly IAuthorizationService _authorizationService;
-        private readonly IChessService _chessService;
-        public ShutdownCommand(IAuthorizationService authorizationService, IChessService chessService) : base(authorizationService)
-        {
-            _authorizationService = authorizationService;
-            _chessService = chessService;
-        }
-
+        public ShutdownCommand()
+        { }
+        [IsAdmin]
         [Command("shutdown")]
```

Comment:
You mind adding a [Summary] to this so it will show up in !help or possibly looking up commands even if they don't have a summary?